ibis/backends/duckdb/__init__.py

```diff
@@ -67,6 +112,24 @@ def do_connect(
         )
         self._meta = sa.MetaData(bind=self.con)
 
+    def register(
+        self,
+        file_name: str | Path,
```

Comment:
```suggestion
        file_name: str | Path,
```

What do you think about `path` here instead?